build/webvr-polyfill.js

```diff
@@ -2604,7 +2604,7 @@ CardboardVRDisplay.prototype.injectPresentModeCssClass_ = function() {
     'left: 0 !important',
     'right: 0 !important',
     'bottom: 0 !important',
-    'z-index: 999999 !important',
+    'z-index: 100 !important',
```

Comment:
I guess if a z-index was needed, it was to be in front of other elements on the page that might have a z-index. So 100 would be big enough to be over these other elements?

Ideally you shouldn't need to set a z-index. Also, a library shouldn't force `!important` styles, it should be overridable by the user of the library.
Maybe the z-index was required for iOS that doesn't implement the fullscreen api?